cmd/controller/controller.go

```diff
@@ -24,8 +24,11 @@ var (
 	configPath    string
 	masterURL     string
 	importerImage string
-	pullPolicy    string
-	verbose       string
+	clonerImage    string
+	importerPullPolicy    string
```

Comment:
The pull policy for either sub component probably doesn't need to be different for each, it would be safe to set it once and apply it to both.